gravatar/src/main/java/com/gravatar/services/HttpException.kt

```diff
@@ -5,11 +5,10 @@ import retrofit2.Response
 /**
  * Exception thrown when an HTTP error occurs in a non-cathing method.
  *
+ * [rawErrorBody] The raw error body of the response.
  * [code] The HTTP status code.
- * [message] The HTTP status message.
  */
-public class HttpException internal constructor(response: Response<*>) : RuntimeException() {
-    internal val rawErrorBody: String? = response.errorBody()?.string()
-    public val code: Int = response.code()
-    public override val message: String = "HTTP ${response.code()} $rawErrorBody"
+internal class HttpException internal constructor(response: Response<*>) : RuntimeException() {
```

Comment:
We'll use this exception only internally to catch HTTP errors.

Reply:
We are throwing it and passing as `originalException`, no? Would it make sense to keep it public for that?

Reply:
Yeah, it makes sense.

Reply:
I think you can keep the constructor private - it's for internal use indeed.

Reply:
I kept it as internal. 👍